README.md

```diff
@@ -1,6 +1,199 @@
 # HMS Continuous Test (CT) Base Infrastructure Repository
 
-This is a MAJOR redisign (v3); go see v1 code for how the RPMs were created.
+This is a MAJOR redesign (v3); go see v1 code for how the RPMs were created.
 This repository contains the docker image `hms-test` that is inherited by the `continuous test` images (eg: `cray-fimrware-action-test` in `hms-firmware-action` repo.
 The image contains pytest, tavern, python, and execution scripts for smoke/functional tests.  This image also includes the default configuration files (that can be overridden).
 This repository also contains a dockerfile for `hms-pytest` which is the legacy way of executing CT rpms.   
+
+
+# How to Test/Develop in a kubernetes environment
+
+This will describe some development tools that have been created to aid in rapid prototyping in a kubernetes environment.
+
+## How to install the development environment
+
+1. Execute this script from an NCN in a tmp directory:
+
+```setup_cray-hms-test-development.sh
+#!/usr/bin/env bash
+#download the chart from artifactory. The raw source is: https://github.com/Cray-HPE/hms-test-charts/tree/main/charts/v1.0/cray-hms-test-development
+wget https://artifactory.algol60.net/artifactory/csm-helm-charts/stable/cray-hms-test-development/cray-hms-test-development-1.0.0.tgz
+
+#upload the hms-test image to the system
+export REMOTE_IMAGE=artifactory.algol60.net/csm-docker/stable/hms-test:3.0.0
+export LOCAL_IMAGE=hms-test:3.0.0
+
+NEXUS_USERNAME="$(kubectl -n nexus get secret nexus-admin-credential --template {{.data.username}} | base64 -d)"
+NEXUS_PASSWORD="$(kubectl -n nexus get secret nexus-admin-credential --template {{.data.password}} | base64 -d)"
+
+podman run --rm --network host quay.io/skopeo/stable \
+    copy --dest-tls-verify=false docker://${REMOTE_IMAGE} docker://registry.local/csm-docker/stable/${LOCAL_IMAGE} \
+    --dest-username "$NEXUS_USERNAME" \
+    --dest-password "$NEXUS_PASSWORD"
+
+#use helm to upgrade/install the chart
+helm upgrade --install -n services cray-hms-test-development ./cray-hms-test-development-1.0.0.tgz
+```
+
+Output should look something like:
```

Comment:
In case you want example output from a fresh "1st time" install (rather than an upgrade) that users will more likely see:

--2022-03-29 16:54:00--  https://artifactory.algol60.net/artifactory/csm-helm-charts/stable/cray-hms-test-development/cray-hms-test-development-1.0.0.tgz
Resolving artifactory.algol60.net (artifactory.algol60.net)... 34.120.105.219
Connecting to artifactory.algol60.net (artifactory.algol60.net)|34.120.105.219|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 17671 (17K) [application/x-gzip]
Saving to: ‘cray-hms-test-development-1.0.0.tgz’

cray-hms-test-development-1.0.0.tgz                     100%[============================================================================================================================>]  17.26K  --.-KB/s    in 0.01s

2022-03-29 16:54:01 (1.38 MB/s) - ‘cray-hms-test-development-1.0.0.tgz’ saved [17671/17671]

Getting image source signatures
Copying blob sha256:1b01ea1be8aca3fd95873b99083e97482c88374c6d648eef8031dc339c88ce2a
Copying blob sha256:3aa4d0bbde192bfaba75f2d124d8cf2e6de452ae03e55d54105e46b06eb8127e
Copying blob sha256:530d8982cc424f38053f7942ba83843739aeeccc8c70a99fb156a964719292f3
Copying blob sha256:235ceb423e67f551977e1abac28d2eaae26e50a020e0972a28ca4b8d897178be
Copying blob sha256:38f8e7472ea57428cc374eba8974cd1e340f69cb9fbba58bc036b8f9e6d52ab3
Copying blob sha256:defb6b2bb024703a6d221086b7718a0ed2801058ca1ee9ca4305e3c2d590c647
Copying blob sha256:f0548e2cebadd91eb72e7ed4be37462f87973ebb11717fd739a28f6be63161c5
Copying blob sha256:00f71969241bae02a4c5a8382fb32a6634e142e6cc780f6a9a21dbdceb77ce7f
Copying blob sha256:b31b23df732d346b415e55c50f13261f5767ff01ad54abdf5dadc5a2cff46e01
Copying blob sha256:4f4fb700ef54461cfa02571ae0db9a0dc1e0cdb5577484a6d75e68dc38e8acc1
Copying blob sha256:2ac4b606308fae3b514bfadcb7b1a9b5666e8fa13849d568e1a85d2907ff7137
Copying config sha256:4c881529d3e1d1dcf95c20645b2310571873bc0519278bd61c260fa3064ae6e6
Writing manifest to image destination
Storing signatures
Release "cray-hms-test-development" does not exist. Installing it now.
NAME: cray-hms-test-development
LAST DEPLOYED: Tue Mar 29 16:54:07 2022
NAMESPACE: services
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
Installation info for chart cray-hms-test-development: